ext/kubernetes/deployment/init.go

```diff
@@ -1,7 +1,9 @@
 package deployment
 
 import (
+	"bytes"
 	"fmt"
+	"html/template"
```

Comment:
Could we actually use text/template instead of html/template?

Reply:
Yes, and we should, I guess. 👍